Recipes.Tests/WebApiTelemetryTests.cs

```diff
@@ -138,16 +140,36 @@ public void TearDown()
             }
 
             telemetryEvents.Single()
-                           .IsIncomingRequest()
-                           .Should()
-                           .BeTrue();
-            telemetryEvents.Single()
                            .CallerIpAddress()
                            .Should()
                            .Be("123.123.123.123");
         }
 
         [Test]
+        public async Task Telemetry_events_based_on_HTTP_request_are_marked_as_incoming()
+        {

```

Comment:
Should this be understood to be a different behavior from when it's based on a response object, e.g. what about outgoing calls?